### PR TITLE
Add generator brazilian phone number

### PR DESCRIPTION
**Pull request by NetonD**
Add following categories to portuguese brazilian:

- Phone

### CODE REVIEW

**Comment by NetonD** [comment]
Is my first contribuition to an open source project, sorry for this, but I need help to undestain what is wrong.

File: lib/faker/phone/pt_br.ex [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    |> number_with_region()
+  end
+
+  @spec number_with_region(binary) :: binary
+  def number_with_region(number) do
+    region = generate_region_code()
+
+    number
+    |> String.replace("xx", region)
+  end
```
Comment (anthonator) [inline lib/faker/phone/pt_br.ex:35]: Could you provide some documentation with examples for `number_with_region/1`?

**Comment by anthonator** [comment]
Thank you for contributing! Could you please update `CHANGELOG.md`?

File: lib/faker/phone/pt_br.ex [hunk 2]
```diff
@@ -0,0 +1,108 @@
+defmodule Faker.Phone.PtBr do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Function to generate brazilians phone  number.
```
Comment (vbrazo) [inline lib/faker/phone/pt_br.ex:5]:
```suggestion
  Function to generate brazilians phone number.
```

Comment (anthonator) [reply lib/faker/phone/pt_br.ex:5]: This still needs to be fixed.

**Review by anthonator — CHANGES_REQUESTED** [review]

**Review by vbrazo — APPROVED** [review]
LGTM

**Review by anthonator — APPROVED** [review]

**Pull request merged**